tmail-backend/apps/distributed-es6-backport/src/main/conf/rabbitmq.properties

```diff
@@ -61,3 +61,11 @@ mailqueue.size.metricsEnabled=false
 # Disable with caution (this only makes sense in a distributed setup where other nodes consume tasks).
 # Defaults to true.
 task.consumption.enabled=true
+
+# AMQP resources parameters for the subscriber email address contact messages. In order to synchronize contacts between Tmail backend and 3rd.
+# Exchange name
+address.contact.exchange=AddressContactExchange1
+# Queue name
+address.contact.queue=AddressContactQueue1
+# Routing key
+address.contact.routingKey=AddressContactRoutingKey1
```

Comment:
We should also have an optional `address.contact.url` stuff: the rabbitMQ used to communicate with contact app might be different than the James values.